src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.WebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)